lib/ae_mdw_web/controllers/tx_controller.ex

```diff
@@ -175,6 +177,24 @@ defmodule AeMdwWeb.TxController do
     end
   end
 
+  @spec pending_txs(Conn.t(), map()) :: Conn.t()
+  def pending_txs(%Conn{assigns: assigns} = conn, _params) do
+    %{state: _state, pagination: pagination, cursor: cursor, scope: scope} =
+      assigns
+
+    node_state = State.new(NodeStore.new())
+
+    with {:ok, paginated_txs} <-
+           Txs.fetch_pending_txs(node_state, pagination, scope, cursor, render_v3?: true) do
+      WebUtil.render(conn, paginated_txs)
+    end
+  end
+
+  @spec pending_txs_count(Conn.t(), map()) :: Conn.t()
+  def pending_txs_count(%Conn{} = conn, _params) do
+    format_json(conn, :aec_tx_pool.size())
```

Comment:
We should probably put this function in the `Node` module, so that it is more easily switchable in the future.